docs/sphinx/source/user_guide/faq.rst

```diff
@@ -138,3 +138,24 @@ so you can check the tables there for more up-to-date tables.
 For modules, if even the SAM files don't include the module you're looking for
 either, you can calculate CEC module model parameters from
 datasheet information using :py:func:`pvlib.ivtools.sdm.fit_cec_sam`.
+
+
+Which should I use, the CEC or the Sandia PV Module database?
+-------------------------------------------------------------
+
+The CEC PV module database contains parameters for significantly more
+modules, and is more up to date, than the Sandia PV module database.
+Therefore, the CEC PV module database is probably the more useful option
+in most cases.  However, finding parameters for the specific module
+being used is more important than which database they came from.
```

Comment:
Also, another distinction between the coefficients is that the CEC PV module database provides parameters for the [SAM/CEC 6-param model](https://www.nrel.gov/docs/fy15osti/64102.pdf) which yields an continuous IV curve whereas the Sandia PV module database provides parameters for the [SAPM](https://energy.sandia.gov/wp-content/gallery/uploads/SAND-2004_PV-Performance-Array-Model.pdf) which is a point model, yielding only max power point, open circuit, short circuit, and two optional intermediate points. Therefore, although both models are useful for finding the max power point, the SAM/CEC 6-param model could be more appropriate when exploration of off-MPP operating conditions is of interest, and the SAPM model could be more useful when just want max power is desired due to it's less computationally expensive yet sufficiently accurate algorithm.

### SAM/CEC 6-param model
![image](https://user-images.githubusercontent.com/1385621/218583372-252fb726-a21d-42ad-baeb-762e0b18af09.png)

### SAPM model
![image](https://user-images.githubusercontent.com/1385621/218583738-e348f45d-d9b7-46e0-b3e0-ebc2d3e1f713.png)